tests/ci/integration/run_s2n_integration.sh

```diff
@@ -25,7 +25,7 @@ S2N_TLS_BUILD_FOLDER="${SCRATCH_FOLDER}/s2n-tls-build"
 
 # Make script execution idempotent.
 mkdir -p ${SCRATCH_FOLDER}
-rm -rf ${SCRATCH_FOLDER}/*
+rm -rf "${SCRATCH_FOLDER:?}"/*
```

Comment:
is it worth pulling this "logic" up into an alias or function in `common_posix_setup.sh` and adding a descriptive error message (i.e. `word` in the bash documentation)?